package.json

```
@@ -41,6 +42,7 @@
     "set-value": "3.0.2",
     "source-map-explorer": "2.5.2",
     "stackdriver-errors-js": "0.8.0",
+    "usa-states": "^0.0.5",
```

nit: use exact versioning to fix the import to what we know works:
```suggestion
    "usa-states": "0.0.5",
```